database/migrations/2022_11_26_170325_create_tax_year_events_table.php

```diff
@@ -13,6 +13,7 @@
     public function up()
     {
         Schema::create('tax_year_events', function (Blueprint $table) {
+            $table->id()->unsigned();
```

Comment:
ℹ️ Should have been added initially – not sure how I missed that (same for the `share_pooling_events` table below)